pkg/controller/vitessshardreplication/repair_replication.go

```diff
@@ -187,6 +193,21 @@ func (r *ReconcileVitessShard) canRepairReplication(ctx context.Context, vts *pl
 	return foundFixableTablet, nil
 }
 
+func masterRdOnlyShouldBeFixed(vts *planetscalev2.VitessShard) bool {
```

Comment:
Check that the condition exists at some point just to be safe.

Reply:
Can do.

Reply:
Done https://github.com/planetscale/vitess-operator/pull/41/commits/b643bd6815b6c3128cea04b544dfb204c7a9b5dc.